src/test/java/trackitnus/logic/parser/ParserUtilTest.java

```diff
@@ -81,25 +82,25 @@ public void parseName_validValueWithWhitespace_returnsTrimmedName() throws Excep
 
     @Test
     public void parsePhone_null_throwsNullPointerException() {
-        Assert.assertThrows(NullPointerException.class, () -> ParserUtil.parsePhone(null));
+        Assert.assertThrows(NullPointerException.class, () -> ParserUtil.parseOptionalPhone(null));
```

Comment:
Should this be Optional.empty()? Or maybe we should add another test case for that

Reply:
Optional.empty here is actually still valid! null will throw exception!